Pull Request from Alex

ganqiupeng opened the pull request:
reverted some files and modified the UI

ganqiupeng committed:
Revert some files

ganqiupeng committed:
Modified UI

ganqiupeng committed:
Added support for Kinect V1, updating...

DanielMerget commented:
There are still some conflicts with master. You should merge your submission first before I can accept the pull request :-)

ganqiupeng committed:
support three modes now: only V2, only V1, both V1 and V2
For V1, can show images and record color and raw depth data in different resolution .

ganqiupeng committed:
resolved the conflict with ganqiupeng/FaceGrabber

ganqiupeng committed:
now can align depth to color space of Kinect V1

ganqiupeng committed:
Fixed a bug of Playback can't find right path

ganqiupeng commented:
have already merged my pull request with the master, now it should be possible to be accepted.
